### PR TITLE
Updates primeng

### PR DESCRIPTION
**Pull request by Iloveall**
## Description

Fixes #...

## Notes (out of scope, known isues, hints for reviewing code, ...)  (optional)

https://github.com/primefaces/primeng/wiki/Migration-Guide

Need to re-install node packages

## Test cases

- [x] Case 1:
  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/prime-ng-update/en/#/groups/mine)
  3. Then I see 2 type of tables without design changes

- [x] Case 2:
  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/en/#/groups/by-id/672913018859223173;path=52767158366271444/details)
  3. Click on start watch then go to Activities tab then Parcours officiels -> Parcours officiels -> L1S1 Computer science
  4. Then click on Chapter View tab -> users
  5. Then I see table without design changes

- [x] Case 3:
  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/prime-ng-update/en/#/activities/by-id/6681168456453971742;path=4702,1352246428241737349,561969155071897034;parentAttempId=0/edit/advanced-parameters)
  3. Then switch Disable score
  4. Switcher must work as before

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,8 +42,8 @@
     "jschannel": "^1.0.2",
```
Comment (ThomasJuster) [inline package.json:42]: I found a UI regression [here](https://dev.algorea.org/branch/feature/prime-ng-update/en/#/groups/by-id/609422240375608067;path=/details/managers) as usual user:
image

Comment (ThomasJuster) [inline package.json:42]: Still scroll problems, I think they are all related so I'll stop the review here
When watching [Super group of mine (Thomas)](https://dev.algorea.org/branch/feature/prime-ng-update/en/#/groups/by-id/6710944276987033666;path=52767158366271444,672913018859223173/details) and navigating to suggested activity "Blockly basic task" / Progress / Chapter view:

image

**Review by ThomasJuster — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (ThomasJuster) [reply package.json:42]: Resolving comment because it's the current behavior in production and is not related to this PR

Comment (ThomasJuster) [reply package.json:42]: Duplicat of #901

Comment (Iloveall) [reply package.json:42]: Fixed

**Review by ThomasJuster — COMMENTED** [review]
All good for me :heavy_check_mark: 
@smadbe you want to take a look ?

**Comment by smadbe** [comment]
I'll assume you have tested carefully most of screen, I have not much to test that you cannot do.

**Review by smadbe — APPROVED** [review]

**Pull request merged**